src/OrchardCore/OrchardCore.Localization.Core/DataAnnotations.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.Extensions.Localization;
+
+namespace OrchardCore.Localization.Core
+{
+    public class DataAnnotations
+    {
+        private readonly IStringLocalizer<DataAnnotations> S;
+
+        /// <summary>
+        /// Creates a new instance of the <see cref="DataAnnotations"/>.
+        /// </summary>
+        /// <param name="localizer">The <see cref="IStringLocalizer"/>.</param>
+        public DataAnnotations(IStringLocalizer<DataAnnotations> localizer)
+        {
```

Comment:
what are you sneaking in here? ;)

Reply:
This is by mistake, I'm sure he commit this yesterday and forgot to delete it ;)

Reply:
File removed